lib/matobj.gi

```diff
@@ -632,7 +632,13 @@ InstallMethod( RowsOfMatrix,
 InstallOtherMethod( NumberRows, "for a plist matrix",
```

Comment:
```suggestion
InstallOtherMethod( NumberRows, "for a matrix",
```
And then in other methods using `IsMatrixObj`, we could write "for a matrix object"